benchmarks/bench_dsl.cpp

```diff
@@ -12,7 +12,7 @@
 #include "common.hpp"
 
 namespace nnfvcc = NeoN::finiteVolume::cellCentred;
```

Comment:
```suggestion
namespace fvcc = NeoN::finiteVolume::cellCentred;
```
We need to be more consistent with the naming of the namespace. 
We go with either `nnfvcc` or `fvcc`. I would prefer the shorter one.

If we choose `fvcc`, then all the occurrences of `nnfvcc` in the same file should be replaced with `fvcc`.